bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -1809,6 +1810,28 @@ public ServerConfiguration setIsForceGCAllowWhenNoSpace(boolean force) {
         return this;
     }
 
+    /**
+     * Get location index sync data.
+     *
+     * @return true  - sync operate location index,
+     *         false - async operate location index.
+     */
+    public boolean getLocationIndexSyncData() {
```

Comment:
This option is not for every LedgerStorage.
We should reflect this into the name